src/components/InventoryTable/helpers.js

```diff
@@ -31,22 +31,36 @@ export const createRows = (rows = [], columns = [], { actions, expandable, noSys
         }];
     }
 
-    return flatten(rows.map((oneItem, key) => ([{
-        ...oneItem,
-        ...oneItem.children && expandable && { isOpen: !!oneItem.isOpen },
-        cells: buildCells(oneItem, columns, extra),
-        actionProps: {
-            'data-ouia-component-id': `${oneItem.id}-actions-kebab`
-        }
-    }, oneItem.children && expandable && {
-        cells: [
-            {
-                title: typeof oneItem.children === 'function' ? oneItem.children() : oneItem.children
-            }
-        ],
-        parent: key * 2,
-        fullWidth: true
-    }]))).filter(Boolean);
+    return flatten(rows.map((oneItem, key) => {
```

Comment:
I think you can use `flatMap` here

Reply:
I believe flatmap is just syntactical sugar. I am not a fan of touching what is working :). 

_It is identical to a [map()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) followed by a [flat()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat) of depth 1 (arr.map(...args).flat())_  see [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap)